docs/docs/filter-sort.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Filter and Sort
+layout: docs
+---
+Use the `RequestOptions` object to define filtering and sorting criteria for an endpoint,
```

Comment:
Do we just want to talk about filter/sort here, because the RequestOptions is also about setting pageSize/pageNum. This might not be the right place to mention that though